src/microbe_stage/MicrobeStage.cs

```diff
@@ -279,9 +279,11 @@ public override void StartNewGame()
 
     public override void StartMusic()
     {
+        var biome = CurrentGame!.GameWorld.Map.CurrentPatch!.BiomeTemplate;
+
         Jukebox.Instance.PlayCategory(GameWorld.PlayerSpecies is EarlyMulticellularSpecies ?
             "EarlyMulticellularStage" :
```

Comment:
Does this work in early multicellular? Does that music category also have the context specific tracks? What happens if no context matches, then I assume nothing can be played? Does that work correctly when playing multiple tracks at once? I saw a Pause call if no tracks are found but that should be skipped if another simultaneously playing track is found.